projects/packages/videopress/src/client/admin/components/video-row/stats.tsx

```diff
@@ -17,10 +17,12 @@ const Stats = ( {
 
 	return (
 		<div className={ classNames( styles.stats, { [ styles.small ]: isSmall } ) }>
-			<Text aria-disabled={ isSmall ? 'false' : 'true' } component="div">
-				{ privacy }
-			</Text>
-			<Text component="div">{ duration }</Text>
+			{ Boolean( privacy ) && (
+				<Text aria-disabled={ isSmall ? 'false' : 'true' } component="div">
+					{ privacy }
+				</Text>
+			) }
+			{ Boolean( duration ) && <Text component="div">{ duration }</Text> }
 			{ Boolean( plays ) && <Text component="div">{ plays }</Text> }
```

Comment:
If a video has 0 plays, the columns are shifted. `plays != null` would fix this case, but if for some reason one of the values is `null`, the column shift still happens. Maybe an empty `Text` to keep the column?
![Screenshot_2022-08-31_16-34-57](https://user-images.githubusercontent.com/8486249/187767647-fefbce9d-d395-4f06-a69e-5d7f0baa33cc.png)

Reply:
Done: https://github.com/Automattic/jetpack/pull/25968/commits/3a8f2a3df8692cff5fccb67ca3954150c6b773b6